api/prometheus/v1/api_test.go

```diff
@@ -849,15 +867,14 @@ func TestAPIClientDo(t *testing.T) {
 				if err == nil {
 					t.Fatalf("expected error %q but got none", test.expectedErr)
 				}
-				if test.expectedErr.Error() != err.Error() {
-					t.Errorf("unexpected error: want %q, got %q", test.expectedErr, err)
-				}
+				testutil.Equals(t, test.expectedErr.Error(), err.Error(), "unexpected error: want %q, got %q", test.expectedErr, err)
```

Comment:
```suggestion
				testutil.Equals(t, test.expectedErr.Error(), err.Error())
```
no need of a message it is more less the default message from `testutil.Equals` anyway